stylesheets/_config.variables.scss

```diff
@@ -220,8 +220,8 @@ $zindex-tour:                         1000 !default;
 $zindex-footer:                       1020 !default;
 $zindex-tooltip:                      1055 !default;
 $zindex-sticky:                       1060 !default;
-$zindex-dropdown:                     1060 !default;
-$zindex-flash:                        1065 !default;
+$zindex-flash:                        1064 !default;
```

Comment:
This needs a special mention in the release notes, we forever have z-index regressions in CMS so this will need specific testing

Reply:
👍